### ISSUE TITLE
Speed Up Compression

### ISSUE THREAD

**Issue by pathquester** [microsoft/LLMLingua#64]
First of all, thank you for this fantastic project. I was wondering if there are any parameters that help with the speed of the compression, currently using TheBloke/Llama-2-7b-Chat-GPTQ but seems to be slow with default parameters even for text that is not really that long.

**Comment by iofu728** [comment]
Hi @pathquester, thank you for your support of LLMLingua. 

In the current implementation, the latency of quantization models is not significantly different from that of full-precision models; it might even be slightly higher. 

In the future, we plan to support more efficient engines #51. If you are limited by hardware resources, you might consider trying smaller models, such as `ahxt/LiteLlama-460M-1T`, or `phi-2` (which we will support soon).

**Comment by iofu728** [comment]
Hi @pathquester,

Thanks to the efforts of the community, `phi-2` is now available for use in LLMLingua. 

Before using it, please update your transformer to the GitHub version by running `pip install -U git+https://github.com/huggingface/transformers.git`.

**Comment by pathquester** [comment]
Thank you so much for the help! Is the GPTQ version also supported now?

**Comment by iofu728** [comment]
Yeah, you can also try to use the GPTQ version like `TheBloke/phi-2-dpo-GPTQ`.

**Comment by pathquester** [comment]
Will try it soon - one clarification, what is the general quality of the results of a tiny model such as GPT2 small for this? Are the results decent and comparable to what is shown in your examples?

**Comment by iofu728** [comment]
Hi @pathquester, based on our experience, even GPT2-small can achieve satisfactory results with moderate compression rates.
